frontend/javascripts/admin/auth/registration_form_wkorg.tsx

```diff
@@ -155,32 +157,57 @@ function RegistrationFormWKOrg(props: Props) {
           placeholder="Password"
         />
       </FormItem>
+      <div className="registration-form-checkboxes">
+        <FormItem
+          name="privacy_check"
+          valuePropName="checked"
+          rules={[
+            {
+              validator: (_, value) =>
+                value
+                  ? Promise.resolve()
+                  : Promise.reject(new Error(messages["auth.privacy_check_required"])),
+            },
+          ]}
+        >
+          <Checkbox>
+            I agree to storage and processing of my personal data as described in the{" "}
+            <a target="_blank" href="/privacy" rel="noopener noreferrer">
+              privacy statement
+            </a>
+            .
+          </Checkbox>
+        </FormItem>
 
-      <FormItem
-        name="privacy_check"
-        valuePropName="checked"
-        rules={[
-          {
-            validator: (_, value) =>
-              value
-                ? Promise.resolve()
-                : Promise.reject(new Error(messages["auth.privacy_check_required"])),
-          },
-        ]}
-      >
-        <Checkbox>
-          I agree to storage and processing of my personal data as described in the{" "}
-          <a target="_blank" href="/privacy" rel="noopener noreferrer">
-            privacy statement
-          </a>
-          .
-        </Checkbox>
-      </FormItem>
-      <FormItem
-        style={{
-          marginBottom: 10,
-        }}
-      >
+        {terms != null && !terms.enabled ? null : (
```

Comment:
I'd also prefer the inverted logic here

Reply:
youre so right, I had difficulty understanding it myself right now :sweat_smile: